<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QLabel" name="state">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>361</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_load">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>load ...</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>471</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>managerExample</string>
    </property>
    <widget class="QLabel" name="Mpara1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>para1</string>
     </property>
    </widget>
    <widget class="QLabel" name="Mpara2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>421</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>para2</string>
     </property>
    </widget>
    <widget class="QLabel" name="Mpara3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>para3</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>471</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>deviceExample 0</string>
    </property>
    <widget class="QLabel" name="Dpara1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>para1</string>
     </property>
    </widget>
    <widget class="QLabel" name="Dpara2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>para2</string>
     </property>
    </widget>
    <widget class="QLabel" name="Dpara3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>para3</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
